### PR TITLE
refactor(lib/netconf): remove bools from netconf

### PR DESCRIPTION
**Pull request by corverroos**
Next step in netconf refactor: remove `IsX` fields.

task: none

### CODE REVIEW

File: e2e/test/e2e_test.go [hunk 1]
```diff
@@ -127,15 +127,17 @@ func test(t *testing.T, testFunc testFunc) {
...
 		for _, chain := range network.Chains {
-			if chain.IsOmniEVM {
-				client, err := ethclient.Dial(chain.Name, chain.RPCURL)
-				require.NoError(t, err)
-
-				t.Run(chain.Name, func(t *testing.T) {
-					t.Parallel()
-					testFunc.TestOmniEVM(t, client)
-				})
+			if chain.ID != network.ID.Static().OmniExecutionChainID {
```
Comment (idea404) [inline e2e/test/e2e_test.go:130]: Why not use the helper here?

e.g. 

`if !(netconf.IsOmniConsensus(network.ID, chain.ID)) {...}`

File: lib/netconf/netconf.go [hunk 2]
```diff
@@ -72,7 +72,7 @@ func (n Network) ChainNamesByIDs() map[uint64]string {
 // OmniEVMChain returns the Omni execution chain config or false if it does not exist.
 func (n Network) OmniEVMChain() (Chain, bool) {
 	for _, chain := range n.Chains {
-		if chain.IsOmniEVM {
+		if chain.ID == n.ID.Static().OmniExecutionChainID {
```
Comment (idea404) [inline lib/netconf/netconf.go:75]: idem, helper here?

File: lib/netconf/static.go [hunk 3]
```diff
@@ -135,3 +135,8 @@ func ConsensusChainIDStr2Uint64(id string) (uint64, error) {
 
 	return resp, nil
 }
+
+// IsOmniConsensus returns true if provided chainID is the omni consensus chain for the network.
+func IsOmniConsensus(network ID, chainID uint64) bool {
```
Comment (idea404) [inline lib/netconf/static.go:140]: Matter of preference but I would prefer to define this as:

`func IsOmniConsensus(network network, chainID chain) bool { ... }`

That way it is always called the same way throughout the codebase

**Review by idea404 — APPROVED** [review]
LGTM. Some comments

File: e2e/test/e2e_test.go (hunk 1 above)
Comment (corverroos) [reply e2e/test/e2e_test.go:130]: I only added the helper when used more than 3 times

File: lib/netconf/static.go (hunk 3 above)
Comment (corverroos) [reply lib/netconf/static.go:140]: one doesn't always have a network, sometimes you only have ID

**Pull request merged**